src/platforms/javascript/guides/electron/index.mdx

```diff
@@ -11,7 +11,7 @@ redirect_from:
 description: "Learn how to use Sentry with Electron."
 ---
 
-`@sentry/electron` is the official Sentry SDK for Electron applications. It can capture JavaScript exceptions in the main process and renderers, as well as collect native crash reports (Minidumps).
+`@sentry/electron` is the official Sentry SDK for Electron applications. It can capture JavaScript exceptions in the `main` and `renderer` processes, as well as collect native crash reports (Minidumps).
```

Comment:
Probably not as it will not be relevant the vast majority of users. 

Manual preload configuration is only required for Electron < v5 when the main process code is bundled. Even in that case, they will see [quite a useful error](https://github.com/getsentry/sentry-electron/blob/master/src/renderer/ipc.ts#L17-L20) to help them debug potential issues.

My concern is that if we mention it here, users are going to try and configure preload scripts and we removed the need for them specifically because it can be so complicated.

Reply:
That’s good perspective, sounds good.

Reply:
I'll ensure that there's an appropriate anchor/section in the docs that we can link to from that error message!